### PR TITLE
Sahil2

### PR DESCRIPTION
**Pull request by SahilKalra98**

### CODE REVIEW

File: src/containers/Home.js [hunk 1]
```diff
@@ -28,6 +28,8 @@ const generateHomeContainer = connect((state) => {
       dispatch({ type: 'CHANGE_WORD', word });
     },
     setOptions: (languageOption, optionValue) => {
+      console.log('language option', languageOption);
```
Comment (dsope05) [inline src/containers/Home.js]: remove console.logs

File: src/components/App.js [hunk 2]
```diff
@@ -49,41 +42,83 @@ const App = ({options, selectedLanguage, changeLanguage, word, changeWord, langu
...
         </div>)
     } else {
-      optionBulletForm.push(<RadioButtonOptions key={title} options={options[chosenType][title]} setOptions={setOptions} 
-      selectedOption={info[title]}
-      title={title}/>);
+      optionBulletForm.push(
+      <div key={title} className="option">
+        <div className="option-title">{title}</div>
+        <div className="option-menu">
+        <Dropdown key={title} options={options[chosenType][title]} setOptions={setOptions}
```
Comment (dsope05) [inline src/components/App.js]: indent

**Review by dsope05 — APPROVED** [review]

**Review by yjlim5 — APPROVED** [review]

File: src/components/App.js [hunk 3]
```diff
@@ -49,41 +42,83 @@ const App = ({options, selectedLanguage, changeLanguage, word, changeWord, langu
...
-
+  let languageButton = [];
+  let nativeBackground = '';
+  for(let i = 0; i < languageList.length; i++){
+    let styleName = 'languages';
+    if(selectedLanguage === languageList[i]){
+      nativeBackground = nativeLanguage[languageList[i]];
+      styleName = 'selectedlanguage';
+      languageButton.push(
+        <span>
```
Comment (songz) [inline src/components/App.js]: y span container?

File: src/components/App.js [hunk 4]
```diff
@@ -49,41 +42,83 @@ const App = ({options, selectedLanguage, changeLanguage, word, changeWord, langu
...
+          <button
+            key={i}
+            className={styleName}
+            onClick={() => {changeLanguage(languageList[i]);}}>
+            <div className='nativeText'>{nativeLanguage[languageList[i]]}</div>
```
Comment (songz) [inline src/components/App.js]: if this className is the only difference between the two, you should make it a variable instead so you can avoid repeating the button component (code dup)

File: src/components/App.js [hunk 5]
```diff
@@ -49,41 +42,83 @@ const App = ({options, selectedLanguage, changeLanguage, word, changeWord, langu
...
-          <img className="conjugate-button"
-            src={myImage}
-            width="73" height="auto" onClick={ submit } />
+        <div className="title-container">
+          <span className="title" >CONJUGATOR</span>
         </div>
-        <input className="output-box" type="textbox" name="output" value={conjugatedWord} />
-      </div>  
+        <div className="title-underline">
+          <img src="http://i.imgur.com/JQNkayi.png?3"/>
```
Comment (songz) [inline src/components/App.js]: download the image and commit it. Don't want to rely on external hosting.

File: src/reducers.js [hunk 6]
```diff
@@ -36,7 +36,13 @@ const appState = (state = {
       return Object.assign({}, state, { conjugatedWord });
     }
     case 'CLEAR_INPUTS': {
-      return Object.assign({}, state, { word: '', conjugatedWord: '' });
+      const newInfo = Object.assign({}, state.info);
+      for(let key in newInfo){
```
Comment (songz) [inline src/reducers.js]: Loop through an object: `Object.keys(newInfo).forEach((key) => { ... });`

File: src/style/App.css [hunk 7]
```diff
@@ -12,32 +12,51 @@ body {
...
-  width: 100%;
-  height: 35px;
+  width: 80%;
+  height: 100%;
   color: black;
-  margin-bottom: 20px;
+  border: transparent;
+  left: 0px;
+  font-size: 18px;
+  position: absolute;
```
Comment (songz) [inline src/style/App.css]: y position absolute? By default, input-box should already be on the left

File: src/components/App.js [hunk 8]
```diff
@@ -2,38 +2,31 @@ import React from 'react';
...
+          options={Object.keys(options)}
+          displayTitle='Word Type'
+          value={info.wordType}
+          onChange={(wordType) => {
+          setOptions('wordType',wordType.value)}}
+        />
+      </div>
+    </div>
   let chosenType = info.wordType || 'verb';
   for(let title in options[chosenType]) {
```
Comment (songz) [inline src/components/App.js]: `options[chosenType].forEach( (title) => {... });`

File: src/components/App.js [hunk 9]
```diff
@@ -49,41 +42,84 @@ const App = ({options, selectedLanguage, changeLanguage, word, changeWord, langu
...
+            value={info[title]}
+          />
+        </div>
+      </div>)
+      ;
     }}
-
+  let languageButton = [];
+  let nativeBackground = '';
+  for(let i = 0; i < languageList.length; i++){
```
Comment (songz) [inline src/components/App.js]: `languageList.forEach( (language, i) => { ...} );`

File: src/components/App.js [hunk 10]
```diff
@@ -49,41 +42,84 @@ const App = ({options, selectedLanguage, changeLanguage, word, changeWord, langu
...
+    let styleName = 'languages';
+    if(selectedLanguage === languageList[i]){
+      nativeBackground = nativeLanguage[languageList[i]];
+      styleName = 'selectedlanguage';
+      languageButton.push(
```
Comment (songz) [inline src/components/App.js]: `textClass = ...`

Comment (songz) [reply src/components/App.js]: `<div className={textClass} ...>`

File: src/components/App.js [hunk 11]
```diff
@@ -49,41 +42,84 @@ const App = ({options, selectedLanguage, changeLanguage, word, changeWord, langu
...
+          <input
+            className="input-box"
+            type="text"
+            name="word"
+            placeholder="Please input your word!"
+            value={word}
+            onChange={(e) => {changeWord(e.target.value);}}
+          />
+          <button
+            type="button" onClick={() => {submit()}} className="btn btn-xs">Conjugate!</button>
```
Comment (songz) [inline src/components/App.js]: `onClick={submit}`

File: src/components/App.js [hunk 12]
```diff
@@ -49,41 +42,79 @@ const App = ({options, selectedLanguage, changeLanguage, word, changeWord, langu
...
-          <img className="conjugate-button"
-            src={myImage}
-            width="73" height="auto" onClick={ submit } />
+        <div className="title-container">
+          <span className="title" >CONJUGATOR</span>
         </div>
-        <input className="output-box" type="textbox" name="output" value={conjugatedWord} />
-      </div>  
+        <div className="title-underline">
+          <img src={myImage}/>
```
Comment (dsope05) [inline src/components/App.js]: better naming than myImage....titleUnderline

**Review by dsope05 — APPROVED** [review]

File: src/components/App.js [hunk 13]
```diff
@@ -2,40 +2,33 @@ import React from 'react';
...
   let wordTypeForm = '';
   let optionListForm = [];
   let optionBulletForm = [];
+  let nativeLanguage = {
+    'Korean': '한국어',
+    'Hindi': 'हिंदी',
+    'French': 'Français',
+    'Vietnamese': 'Tiếng Việt'
+  }
   wordTypeForm =
```
Comment (songz) [inline src/components/App.js]: `let wordTypeForm = ...`

File: src/components/App.js [hunk 14]
```diff
@@ -49,41 +42,79 @@ const App = ({options, selectedLanguage, changeLanguage, word, changeWord, langu
...
+  languageButton = languageList.map( (language, i) => {
+  let styleName = 'languages';
+  let textClass = '';
+  if(selectedLanguage === language){
+    nativeBackground = nativeLanguage[language];
+    styleName = 'selectedlanguage';
+    textClass = 'nativeText';
+  }
+  return(
+    languageButtonClass=
```
Comment (songz) [inline src/components/App.js]: what is going on here?

File: src/components/App.js [hunk 15]
```diff
@@ -49,41 +42,79 @@ const App = ({options, selectedLanguage, changeLanguage, word, changeWord, langu
...
+          />
+        </div>
+      </div>)
+      ;
+    }});
+  let languageButton = [];
+  let nativeBackground = '';
+  let languageButtonClass = '';
+  languageButton = languageList.map( (language, i) => {
+  let styleName = 'languages';
```
Comment (songz) [inline src/components/App.js]: indent

File: src/style/App.css [hunk 16]
```diff
@@ -12,32 +12,50 @@ body {
...
   padding-left: 20px;
   padding-right: 20px;
   font-size: 3em;
-  font-family: 'Archivo Black', sans-serif;
+  font-family: "Myriad Pro", "Gill Sans", "Gill Sans MT", Calibri, sans-serif;
   font-weight: bolder;
 }
 
+.title-underline {
+  position: absolute;
```
Comment (songz) [inline src/style/App.css]: don't absolute position this. Try to stretch out your page wide and thin.

File: src/components/App.js [hunk 17]
```diff
@@ -49,41 +41,77 @@ const App = ({options, selectedLanguage, changeLanguage, word, changeWord, langu
...
     } else {
-      optionBulletForm.push(<RadioButtonOptions key={title} options={options[chosenType][title]} setOptions={setOptions} 
-      selectedOption={info[title]}
-      title={title}/>);
-    }}
-
+      optionBulletForm.push(
+      <div key={title} className="option">
+        <div className="option-title">{title}</div>
+        <div className="option-menu">
```
Comment (songz) [inline src/components/App.js:47]: could we try putting option-menu classname directly into the Dropdown component?

File: src/components/App.js [hunk 18]
```diff
@@ -49,41 +41,77 @@ const App = ({options, selectedLanguage, changeLanguage, word, changeWord, langu
...
-          <img className="conjugate-button"
-            src={myImage}
-            width="73" height="auto" onClick={ submit } />
+        <div className="title-container">
+          <span className="title" >CONJUGATOR</span>
         </div>
-        <input className="output-box" type="textbox" name="output" value={conjugatedWord} />
-      </div>  
+        <div className="title-underline">
+          <img src={titleUnderline}/>
```
Comment (songz) [inline src/components/App.js]: can img simply be inside title-container

**Pull request merged**